src/components/inputBar.tsx

```diff
@@ -418,20 +437,41 @@ const InputBar = (props: InputBarProps) => {
               animate={{ x: 0, opacity: 1, transition: { duration: 0.5 } }}
               exit={{ x: 50, opacity: 0, transition: { duration: 0.5 } }}
             >
-              <Button
-                size="icon"
-                variant="secondary"
-                disabled={
-                  props.isChatCompleted ||
-                  isRecording ||
-                  isTranscribing ||
-                  disableInputs
-                }
-                type="submit"
-                className="disabled:text-muted"
-              >
-                <PaperPlaneTilt className="h-4 w-4 fill-current" />
-              </Button>
+              {props.dropZone ? (
+                <>
+                  <Button
+                    size="icon"
+                    variant="secondary"
+                    disabled={
+                      props.isChatCompleted ||
+                      isRecording ||
+                      isTranscribing ||
+                      disableInputs
+                    }
+                    onClick={handleDropzone}
+                    className="disabled:text-muted"
+                  >
+                    <PaperPlaneTilt className="h-4 w-4 fill-current" />
+                  </Button>
+                </>
+              ) : (
+                <>
+                  <Button
+                    size="icon"
+                    variant="secondary"
+                    disabled={
+                      props.isChatCompleted ||
+                      isRecording ||
+                      isTranscribing ||
+                      disableInputs
+                    }
+                    type="submit"
+                    className="disabled:text-muted"
+                  >
+                    <PaperPlaneTilt className="h-4 w-4 fill-current" />
+                  </Button>
+                </>
+              )}
```

Comment:
having only one button can do the thing,  no need to have two different buttons rendering conditionally

Reply:
Resolved it